CONTRIBUTING.adoc

```diff
@@ -0,0 +1,27 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/minishift/minishift-addons/issues[Github issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+.  Create an issue describing your proposed change to the repository.
+.  The repository owners will triage and respond to your issue promptly.
+.  Fork the repository and create a topic branch.
+.  Submit a pull request with the proposed changes.
+
+NOTE: If you submit a new add-on, the add-on must be added as a sub-directory in the root of the repository.
+See the documentation for information about link:https://docs.openshift.org/latest/minishift/using/addons.html#add-ons-overview[file structure and content format].
+
+[[questions]]
+== Questions?
+
+If you run into issues or have any questions about contributions, feel free to ping the Minishift developers on IRC at the #minishift channel on Freenode.
```

Comment:
@thatdocslady I just wanted to make sure we mention the mailing list along with IRC.